Author: Andreas Tille <tille@debian.org>
LastChanged: Wed, 23 Jan 2013 17:04:13 +0100
Description: Try hard to get CXXFLAGS into build to enable hardening
 CXXFLAGS are hardwired inside configure.in so simply setting the
 environment variable does not help here and we need to recreate
 configure script using dh_autoreconf

--- a/configure.in
+++ b/configure.in
@@ -33,7 +33,7 @@ ACX_MPI
 # Set compiler and compiler flags
 CXX="$MPICXX"
 CFLAGS="-g -O0"
-CXXFLAGS="-g -O0 -ansi -pedantic -Wall -Wextra -Wno-long-long -Wno-unused-parameter -DHAVE_MPI_CPP -lm"
+CXXFLAGS="-g -O0 -ansi -pedantic -Wall -Wextra -Wno-long-long -Wno-unused-parameter -DHAVE_MPI_CPP -lm ${CXXFLAGS}"
 
 # Autoconf output
 AC_CONFIG_FILES([
